packages/components/src/Filter/FilterSelect/FilterSelect.tsx

```diff
@@ -93,7 +94,7 @@ export const FilterSelect = <Option extends SelectOption = SelectOption>({
           })
         }
         onMount={setTriggerRef}
-        classNameOverride={classNameOverride}
+        classNameOverride={classnames(styles.filterSelect, classNameOverride)}
```

Comment:
This `filterSelect` class doesn't seem to exist in CSS

Reply:
Ah true, this is when I was testing to see if this something tethered this to a higher specificity. Will axe it since it's not in use

Reply:
Removed and good for another squiz 👀  @dougmacknz